bin/server.dart

```diff
@@ -16,6 +16,7 @@ Future<void> main() async {
     o.release = Platform.environment['VERSION'];
     o.environment = Platform.environment['ENVIRONMENT'];
     o.debug = !isProduction;
+    o.enablePrintBreadcrumbs = false;
```

Comment:
There are quite a lot of print statements but they don't seem like they should be recorded.